src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -101,7 +101,7 @@ public async Task<ActionResult<IEnumerable<DataFileInfo>>> GetDataFileInfo(Guid
         [RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = int.MaxValue)]
         public async Task<ActionResult<DataFileInfo>> AddDataFilesAsync(Guid releaseId,
             [FromQuery(Name = "replacingFileId")] Guid? replacingFileId,
-            [FromQuery(Name = "name")] string subjectName,
+            [FromQuery(Name = "title")] string subjectName,
```

Comment:
Is there a reason we're doing this manually like this and not being handed a `AddDataFileViewModel` or similar, as we do in other places?  Like we currently use `ReleaseAncillaryFileUploadViewModel`?  I'm aware this is pre-existing code and not something you've added - just wondering if there was a reason...

Reply:
Oh, I've answered my own question - query params.  Which changes my question to, is there any preference in using query params in POSTs over a request body?  Understandably we need the body wherever we upload Files, but just personal preference for me is to use a body all over the place.

Anyway, nowt to do with this PR, so forget it :)

Reply:
Yep, so I think this is just technical debt from when this was originally written. We really should be binding it to a model using the `[FromForm]` annotation like we've done in the ancillary file upload for this PR). 

I decided against doing this as part of this ticket as there's a bunch of things to clean up here :smile:

I've created this ticket for it: https://dfedigital.atlassian.net/browse/EES-2521

Reply:
Yup understandable, I knew it was out of the remit of this PR, but thanks for creating the ticket!